src/main/java/com/alkemy/ong/controller/v1/OrganizationController.java

```diff
@@ -22,4 +22,12 @@
         return ResponseEntity.ok().body(service.getOrganization(id));
     }
 
+    @PutMapping("/public/{id}")
+    public ResponseEntity<?> updateOrganization(@PathVariable Long id, @Valid @RequestBody OrganizationPutDto organization, BindingResult result){
+        if (result.hasErrors())
```

Comment:
Please do not use `BindingResult` result as validation strategy, use @Valid instead.